components/ModSubmissionsEditFormFacility.vue

```diff
@@ -13,44 +13,49 @@
                 type="text"
                 :placeholder="$t('modSubmissionForm.placeholderTextFacilityNameEn')"
                 :required="true"
-                :input-validation-check="validateNameEn"
-                :invalid-input-error-message="$t('modSubmissionForm.inputErrorMessageFacilityNameEn')"
+                :inputValidationCheck="validateNameEn"
```

Comment:
change the prop to: `input-validation-check`
I think it's a linting warning and not error, but since we are also using the `data-testid` I think it's more consistent. I think running yarn lint:fix will do this for you.